formspree/templates/users/account.html

```diff
@@ -52,51 +52,51 @@ <h3>Plan</h3>
-                    </div>
-                </td>
-            </tr>
-            {% endfor %}
-          </table>
-          {% else %}
-          <p>We couldn't find any active cards in your wallet. Please make sure to add a card by {{ sub.current_period_end }} or your subscription won't renew.</p>
-          {% endif %}
-          {% include "users/card.html" %}
+{#          <h3>Wallet</h3>#}
+{#          {% if cards %}#}
+{#          <table id="card-list">#}
+{#            {% for card in cards %}#}
+{#            <tr>#}
+{#                <td>#}
+{#                    <div class="arrow"><i class="fa fa-chevron-right" aria-hidden="true"></i></div>#}
+{#                </td>#}
+{#                <td><i class="fa fa-{{card.css_name}}" aria-hidden="true"></i></td>#}
+{#                <td>••••{{ card.last4 }}</td>#}
+{#                <td>{{card.exp_month}}/{{card.exp_year}}</td>#}
+{#            </tr>#}
+{#            <tr>#}
+{#                <td colspan="4">#}
+{#                    <div class="actions" style="float:right;width:50%;padding-left:20%;">#}
+{#                        {% if card.default %}#}
+{#                        <p><button style="color:white;background:#359173;border:none;" class="disabled" disabled>Default</button></p>#}
+{#                          {% else %}#}
+{#                          <form action="{{ url_for('change-default-card', cardid=card.id) }}" method="POST">#}
+{#                            <button type="submit" style="margin-bottom: 18px;">Make Default</button>#}
+{#                          </form>#}
+{#                        {% endif %}#}
+{#                        <form action="{{ url_for('delete-card', cardid=card.id) }}" method="POST">#}
+{#                            <button type="submit">Delete</button>#}
+{#                        </form>#}
+{#                    </div>#}
+{#                    <div class="row">#}
+{#                        <p>Number: ••••{{ card.last4 }}</p>#}
+{#                        <p>Type: {{ card.brand }} {{ card.funding }} card</p>#}
+{#                        <p>Origin: {{ card.country }} <img src="{{url_for('static', filename='img/countries/%s.png' % card.country.lower())}}" width="25"></p>#}
+{#                        <p>CVC Check: {% if card.cvc_check == "pass" %}Passed <i class="fa fa-check-circle-o" aria-hidden="true"></i>{% elif card.cvc_check == "fail"%}Failed <i class="fa fa-times-circle-o" aria-hidden="true"></i>{%else%}Unknown <i class="fa fa-question-circle" aria-hidden="true"></i>{%endif%}</p>#}
+{#                    </div>#}
+{#                </td>#}
+{#            </tr>#}
+{#            {% endfor %}#}
+{#          </table>#}
+{#          {% else %}#}
+{#          <p>We couldn't find any active cards in your wallet. Please make sure to add a card by {{ sub.current_period_end }} or your subscription won't renew.</p>#}
+{#          {% endif %}#}
+{#          {% include "users/card.html" %}#}
```

Comment:
Why not delete these lines instead of commenting them?

Reply:
My bad. Just something I did during development that I will remove